src/main/java/com/sollace/coppergolem/entity/ai/PositionFinder.java

```diff
@@ -69,8 +68,8 @@ private Stream<BlockPos> findPositions() {
 
     protected Stream<BlockPos> searchArea(int range) {
         var stream = BlockPos.streamOutwards(entity.getBlockPos(), range, range, range)
-                .filter(this::isExposed)
-                .filter(p -> entity.getEntityWorld().isChunkLoaded(p) && isValid(toState(p)));
+            .filter(this::isExposed)
+            .filter(p -> entity.getEntityWorld().isChunkLoaded(p.getX(), p.getZ()) && isValid(toState(p)));
```

Comment:
Same as above. If there's a single-parameter version of this method, please use that.